pkg/volume/azure_dd/mounter.go

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
filename prefixed with azure_ for log parsing. 
For new files, copyright year stay with 2017